corehq/apps/reports/standard/deployments.py

```diff
@@ -477,12 +492,46 @@ def _fmt_timestamp(timestamp):
 
         for row in table[1:]:
             # Last submission
-            row[len(location_colums) + 1] = _fmt_timestamp(row[len(location_colums) + 1])
-            # Last sync
             row[len(location_colums) + 2] = _fmt_timestamp(row[len(location_colums) + 2])
+            # Last sync
+            row[len(location_colums) + 3] = _fmt_timestamp(row[len(location_colums) + 3])
         result[0][1] = table
         return result
 
+    def get_location_column(self, user, user_loc_dict):
+        if not user.get('assigned_location_ids'):
+            return '---'
+        return self._get_formatted_assigned_location_names(user, user_loc_dict)
+
+    def _get_formatted_assigned_location_names(self, user, user_loc_dict):
+        """
+        Create an HTML formatted string of the given assigned location names.
+        The primary location will be highlighted in bold.
+        """
+        assigned_location_ids = user.get('assigned_location_ids')
```

Comment:
Hey @zandre-eng 

Do we need to ensure that we only show locations relevant to the domain? are these always only mobile workers or these could be web users as well?

Reply:
That is a good point. It can be web users as well, and so I think we should rather be safe and constrain this to a specific domain.

Reply:
Addressed in [c1a088b](https://github.com/dimagi/commcare-hq/pull/34744/commits/c1a088b0477ad503df92cf40494d1336f279d16a)